### PR TITLE
Updating the Lombard Info

### PR DESCRIPTION
**Pull request by jameswstevens**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):

Lombard

##### Twitter Link:

https://x.com/Lombard_Finance

##### List of audit links if any:

Smart Contracts:
Halborn: [smart contract report](https://www.halborn.com/audits/lombard/lbtc)
Veridise: [smart contract report](https://veridise.com/wp-content/uploads/2024/08/VAR_Lombard_240506.pdf)

Consortium:
Halborn: [consortium report](https://www.halborn.com/audits/lombard/consortium)

Veda Vault: https://github.com/Se7en-Seas/boring-vault/tree/main/audit

##### Website Link:

https://www.lombard.finance/

##### Logo (High resolution, will be shown with rounded borders):

Already have the logo on Defi Llama

##### Current TVL:

$255,880,057M

##### Chain:

BTC -> ETH, more chains coming soon

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)

https://www.coingecko.com/en/coins/lombard-staked-btc

##### Short Description (to be shown on DefiLlama):

Lombard is transforming Bitcoin’s utility from a store of value into a productive financial tool through a security-first liquid Bitcoin primitive—LBTC

##### Token address and ticker if any:

LBTC - 0x8236a87084f8b84306f72007f36f2618a5634494

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Liquid Staking

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/etherfi-lrt exports TVL: 
        
 
 
    ethereum                  435.67 M
    
    total                    435.67 M

**Comment by llamatester** [comment]
The adapter at projects/lombard-liquid exports TVL: 
        
 
 
    ethereum                  255.25 M
    
    total                    255.25 M

**Comment by llamatester** [comment]
The adapter at projects/lombard-vault exports TVL: 
        
 
 
    ethereum                  38.16 M
    
    total                    38.16 M

**Comment by llamatester** [comment]
The adapter at projects/lombard exports TVL: 
        
 
 
    bitcoin                   257.24 M
    
    total                    257.24 M

**Comment by jameswstevens** [comment]
@g1nt0ki: 
1) Can we have an interface like etherfi with our locked BTC and vault on different pages/
2) Update to our name, it shoudl just be Lombard
3) since it is just the org, can we be moved to Liquid Staking since that is our main category
4) updated our description, since this is the org not LBTC

**Comment by llamatester** [comment]
The adapter at projects/etherfi-lrt exports TVL: 
        
 
 
    ethereum                  435.68 M
    
    total                    435.68 M

**Comment by llamatester** [comment]
The adapter at projects/lombard-liquid exports TVL: 
        
 
 
    ethereum                  255.16 M
    
    total                    255.16 M

**Comment by llamatester** [comment]
The adapter at projects/lombard-vault exports TVL: 
        
 
 
    ethereum                  38.17 M
    
    total                    38.17 M

**Comment by llamatester** [comment]
The adapter at projects/lombard exports TVL: 
        
 
 
    bitcoin                   257.58 M
    
    total                    257.58 M

File: projects/lombard-liquid/index.js
[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/lombard-liquid/index.js:1]: *this was meant for lombard-vault listing

For this file, the backing of these tokens are already counted in lomard/inde.js on bitcoin side right?

File: projects/etherfi-lrt/test.js
Comment (g1nt0ki) [inline projects/etherfi-lrt/test.js:1]: you can test tvl of an adapter by running `npm test projects/<adapterName>` 

I am tempted to reject this PR as it bring nothing new

**Comment by g1nt0ki** [comment]
hi @jameswstevens we will update the project name to "Lombard"

**Pull request closed without merging**